client/tests/setup/mocks.ts

```diff
@@ -0,0 +1,21 @@
+// To return a different value for each test, you can import this file and use mockReturnValue
+// to set the return value for the function you want to mock using mockReturnValue
+// or mockReturnValueOnce. Example:
+//
+// import { useRouter } from "@/tests/setup/mocks";
+//
+// useRouter.mockReturnValue({
+//  query: {
+//      formSection: "3",
+//      operation: "test-operation-id",
+//    },
+//    replace: vi.fn(),
+//  });
+
+const actionHandler = vi.fn();
```

Comment:
These are going to be super convenient for mocking; thanks for setting these up!

Reply:
Thanks. This setup looks fairly simple but it took me a lot of time and trial and error to figure out how to handle this and work my way through mocking errors 🙈